src/consumer/mod.rs

```diff
@@ -237,7 +237,7 @@ impl fmt::Debug for RecordJS {
     }
 }
 
-use fluvio::dataplane::ErrorCode;
```

Comment:
Is this change needed? I wonder if we can avoid installing fluvio_protocol and just use the fluvio crate that is supposed to be reexporting these strucs

Reply:
Let me recheck! Probably I didn't noticed the reexport on this case.

Reply:
Okey export is aliased to `dataplate`! I'm updating..

Reply:
Addressed on: [8a84cb6](https://github.com/infinyon/fluvio-client-node/pull/269/commits/8a84cb631e85e96d48b32fbe8714dbd38007a7bb)